src/taoensso/carmine/message_queue.clj

```diff
@@ -106,24 +106,25 @@
                            perform a de-duplication check. If unspecified, a
                            unique id will be auto-generated.
     * allow-requeue?     - When true, allow buffered escrow-requeue for a
-                           message in the :locked or :done-with-backoff state."
-  ;; TODO Option to enqueue something with an init backoff?
+                           message in the :locked or :done-with-backoff state.
+    * initial-backoff    - Initial backoff in millis."
   (let [script (enc/have (enc/slurp-resource "taoensso/carmine/lua/mq/enqueue.lua"))]
-    (fn [qname message & [unique-message-id allow-requeue?]]
+    (fn [qname message & [unique-message-id allow-requeue? initial-backoff]]
```

Comment:
These multiple arity options are getting a bit out of hand.
I'd prefer we switch to `[qname message & [{:keys [unique-message-id allow-requeue? initial-backoff-ms]}]`.

I'll tweak after to ensure backward compatibility.

Reply:
Oh I see - the functions then needs to be called like `(enqueue "q" "m" {:initial-backoff-ms 250})` - I'm not sure how to fix that with backward compatibility but I can put in the keyword args change in this patch if you like. Or maybe that should be a followup?

Reply:
Correct 👍 

Please put in the keyword args change in this patch, and I'll add the backward compatibility myself after.
This way your new tests can already make use of the updated API.